chainer/function_hook.py

```diff
@@ -76,15 +84,24 @@ class FunctionHook(object):
        as a thread local object. So, function hooks registered
        are different depending on threads.
 
+    If the hook is registered in this way, :meth:`~chainer.FunctionHook.added`
```

Comment:
It takes time for me to notice that this sentence is a continuation of the last paragraph above the long example/note sections. How about moving it to the end of the last paragraph?

Reply:
Fixed.